Automattic/jetpack
# Instant Search: remove date from search results

Fixes https://github.com/Automattic/jetpack/issues/14609

#### Changes proposed in this Pull Request:

* Remove date from minimal search results.

#### Before:

![image](https://user-images.githubusercontent.com/390760/74443210-b47fdd80-4e6a-11ea-952c-e0f98aa536e2.png)

#### After:

![image](https://user-images.githubusercontent.com/390760/74443270-ca8d9e00-4e6a-11ea-9157-208ec7a2cc99.png)


#### Is this a new feature or does it add/remove features to an existing part of Jetpack?
* No.

#### Testing instructions:
1. Follow [these setup instructions](https://github.com/Automattic/jetpack/blob/instant-search-master/modules/search/instant-search/README.md#testing-instructions) to set up Jetpack Instant Search on your site.
2. Search your site to trigger the Jetpack Search overlay.
3. Ensure that the date is no longer displayed on search results.

#### Proposed changelog entry for your changes:
* None.
